edib/make-app.mk

```diff
@@ -27,7 +27,7 @@ postinfo:
 release: $(RELEASE_FILE)
 
 $(RELEASE_FILE): app-compile phoenix-assets
-	$(IN_APP_DIR) mix release --env=$(MIX_ENV:-prod)
```

Comment:
Ah, sorry, actually my bad.

What I meant was `--env=${MIX_ENV:-prod}` which would work in a shell session. But in make's case we already have a default value if not set.
(So, good spot!)